gui/wxpython/mapdisp/properties.py

```diff
@@ -277,6 +283,42 @@ def _onToggleCheckBox(self, event):
             self.widget.SetLabel(self.defaultLabel)
 
 
+class RBEnum(Enum):
+    """Radiobutton types and their numbers"""
+
+    coordinates = (1,)
+    mapscale = (2,)
+    compRegion = (3,)
+    goTo = (4,)
+    compExtent = (5,)
+    displayGeometry = 6
```

Comment:
Why tuples here? And the last one is int?

Reply:
Interesting. I did not make it, that had to be caused by Black.